igb_client/client.py

```diff
@@ -0,0 +1,127 @@
+from typing import List, Dict, Optional, Union
+
+import requests_cache
+
+from igb_client.dataclasses import JobBoard, ATSCredential, ContractCredential, Credential
+from igb_client.encrypt import AESCypher
+from igb_client.parse import parse_igb_xml_payload
+
+
+class IGBClientBase:
+    _instance = None
+    _base_url = "{base_url}/{environment_id}/{view}"
+    _credentials_storage_key = None
+
+    def __init__(self, api_key: str, environment_id: str, credentials_storage_key: str,
+                 base_url="https://api.ingoedebanen.nl/apipartner/hapi/v1"):
+        self._environment_id = environment_id
+        self._credentials_storage_key = credentials_storage_key
```

Comment:
What about the credentials _transport_ key?

Reply:
added key and helper methods